components/03-modules/slider/_slider.scss

```diff
@@ -0,0 +1,178 @@
+$slider__margin                  : $spacer--medium !default;
+$slider__item-padding            : $spacer--medium !default;
+$slider__transition              : $transition-base !default;
+$slider__navigation-icon__spacing: 48px !default;
+$slider__name-font-weight        : $font-weight-base !default;
+$slider__name-font-size          : $font-size-base !default;
+
+$slider__price-font-weight       : $font-weight-bold !default;
+$slider__price-color             : $black !default;
+$slider__price-old-color         : $black !default;
+$slider__price-old-font-weight   : $font-weight-base !default;
+$slider__price-old-font-size     : $font-size-extra-small !default;
+$slider__price-new-color         : $color-primary !default;
+$slider__price-new-font-wieght   : $font-weight-bold !default;
+$slider__price-new-font-size     : $font-size-small !default;
+
+$slider__icon-spacing            : 25px !default;
+$slider__icon-fill               : $green !default;
+$slider__icon-fill-hover         : #007025 !default;
+
+.slider {
+    margin-bottom: $slider__margin;
+    position: relative;
+
+    &__title {
+        margin-bottom: $slider__margin * 2;
+        width: 100%;
+    }
+
+    &__handler {
+        overflow: hidden;
+    }
+
+    &__slides {
+        display: flex;
+    }
+
+    &__item {
+        position: relative;
+        padding: 0;
+        min-width: 50%;
+        padding-right: $slider__item-padding;
+        border: 0;
+
+        @include mq($screen-m) {
+            min-width: 33%;
+        }
+
+        @include mq($screen-l) {
+            min-width: 25%;
+        }
+
+        .product__img-box {
+            min-height: unset;
+        }
+
+        .product__type {
+            padding: 0;
+        }
+    }
+
+    &__link {
+        text-decoration: none;
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__image {
+        margin-bottom: $spacer;
+    }
+
+    &__item-handler {
+        display: flex;
+        flex-wrap: wrap;
+
+        @include mq($screen-m) {
+            flex-wrap: nowrap;
+            justify-content: space-between;
+        }
+    }
+
+    &__name {
+        flex-basis: 100%;
+        font-weight: $slider__name-font-weight;
+
+        @include mq($screen-m) {
+            flex-basis: auto;
+            margin-right: $spacer;
+            margin-bottom: 0;
+        }
+
+        @include mq($screen-l) {
+            font-size: $slider__name-font-size;
+        }
+    }
+
+    &__price {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        color: $slider__price-color;
+        font-weight: $slider__price-font-weight;
+
+        @include mq($screen-m) {
+            flex-flow: column;
+            align-items: flex-end;
+            min-width: fit-content;
```

Comment:
I see that `fit-content` is not supported by edge(https://caniuse.com/#search=fit-content)...should we use it ?

Reply:
🤔

Reply:
searched for the other solution and found `white-space` but it's not supported by edge too `https://caniuse.com/#search=white-space` 😫 dunno what to do with that

Reply:
Then use `white-space` property I guess.

Reply:
ok